HelmSetup.md

```diff
@@ -193,7 +193,7 @@ Some useful parameters for the chart, you could also check them in values.yaml
 | ui.securityContext                        | pod security context values                                                           | {}                       |
 | ui.containerSecurityContext               | container security context values                                                     | {}                       |
 | ui.livenessProbe                          | container livenessprobe                                                               | see Values.yaml          |
-| ui.readinessProbe                         | container readinessProbe                                                              | {}                       |
```

Comment:
Do we need to open this readinessProbe and livenessProbe as configurable?
IMO, devlake can define its expected startup time and does not need a user/customer to configure it.

Reply:
hi @matrixji , users should be able to config the time, failure times and so on, I think it makes sense